x-pack/plugins/index_lifecycle_management/public/sections/edit_policy/components/delete_phase/delete_phase.js

```diff
@@ -0,0 +1,120 @@
+import PropTypes from 'prop-types';
+import { MinAgeInput } from '../min_age_input';
+
+import {
+  EuiDescribedFormGroup,
+  EuiButton,
+} from '@elastic/eui';
+import {
+  PHASE_DELETE,
+  PHASE_ENABLED,
+  PHASE_ROLLOVER_MINIMUM_AGE,
+  PHASE_ROLLOVER_MINIMUM_AGE_UNITS,
+} from '../../../../store/constants';
+import { ActiveBadge, PhaseErrorMessage } from '../../../components';
+
+export class DeletePhase extends PureComponent {
+  static propTypes = {
+    setPhaseData: PropTypes.func.isRequired,
+    isShowingErrors: PropTypes.bool.isRequired,
+    errors: PropTypes.object.isRequired,
+    phaseData: PropTypes.shape({
+      [PHASE_ENABLED]: PropTypes.bool.isRequired,
+      [PHASE_ROLLOVER_MINIMUM_AGE]: PropTypes.oneOfType([
+        PropTypes.number,
+        PropTypes.string
+      ]).isRequired,
+      [PHASE_ROLLOVER_MINIMUM_AGE_UNITS]: PropTypes.string.isRequired
+    }).isRequired
+  };
+
+  render() {
+    const {
+      setPhaseData,
+      phaseData,
+      errors,
+      isShowingErrors,
+    } = this.props;
+
+    return (
+      <EuiDescribedFormGroup
+        title={
+          <div>
+            <span className="eui-displayInlineBlock eui-alignMiddle">
+              <FormattedMessage
+                id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.deletePhaseLabel"
+                defaultMessage="Delete phase"
+              />
+            </span>{' '}
+            {phaseData[PHASE_ENABLED] ? (
+              <ActiveBadge />
+            ) : null}
+          </div>
+        }
+        titleSize="s"
+        description={
+          <Fragment>
+            <p>
+              <FormattedMessage
+                id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.deletePhaseDescriptionText"
+                defaultMessage="You no longer need your index.  You can define when it is safe to delete it."
+              />
+
+            </p>
+            {phaseData[PHASE_ENABLED] ? (
+              <EuiButton
+                color="danger"
+                onClick={async () => {
+                  await setPhaseData(PHASE_ENABLED, false);
+                }}
+                aria-controls="deletePhaseContent"
+              >
+                <FormattedMessage
+                  id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.deactivateDeletePhaseButton"
+                  defaultMessage="Deactivate delete phase"
+                />
+              </EuiButton>
+            ) : (
+              <EuiButton
+                onClick={async () => {
+                  await setPhaseData(PHASE_ENABLED, true);
+                }}
+                aria-controls="deletePhaseContent"
+              >
+                <FormattedMessage
+                  id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.activateDeletePhaseButton"
+                  defaultMessage="Activate delete phase"
+                />
+              </EuiButton>
+            )}
+            <PhaseErrorMessage isShowingErrors={isShowingErrors} />
```

Comment:
This error message looks a little tacked on here:

![image](https://user-images.githubusercontent.com/1238659/49312856-54d22800-f49a-11e8-9e3a-fbc9adf441f7.png)

How about replacing the "Active" badge with a "Fix errors" badge? I think this makes sense since this badge represents status, and the status of this phase is that it requires attention. It's also not active if it has errors, so it makes sense to me to switch between them.

![image](https://user-images.githubusercontent.com/1238659/49312888-7206f680-f49a-11e8-9a5b-29a071ddf629.png)

```js
            {phaseData[PHASE_ENABLED] && !isShowingErrors ? (
              <ActiveBadge />
            ) : null}{' '}
            <PhaseErrorMessage isShowingErrors={isShowingErrors} />
```

```js
// phase_error_message.js

import React from 'react';
import { EuiBadge } from '@elastic/eui';
import { FormattedMessage } from '@kbn/i18n/react';
export const PhaseErrorMessage = ({ isShowingErrors }) => {
  return isShowingErrors ? (
    <EuiBadge className="eui-alignMiddle" color="danger">
      <FormattedMessage
        id="xpack.indexLifecycleMgmt.editPolicy.phaseErrorMessage"
        defaultMessage="Fix errors"
      />
    </EuiBadge>
  ) : null;
};
```